nilearn/plotting/tests/test_surf_plotting.py

```diff
@@ -1115,3 +1115,22 @@ def test_compute_facecolors_matplotlib():
             0.5,
             alpha,
         )
+
+
+def test_plot_surf_roi_plotly_sym_map():
+    """Regression test for https://github.com/nilearn/nilearn/issues/3941"""
+    if not PLOTLY_INSTALLED:
+        pytest.skip('Plotly is not installed; required for this test.')
+    mesh, roi_map, _ = _generate_data_test_surf_roi()
+    # plot roi
+    vmax = 0.4
+    vmin = -0.4
+    cmap = "RdYlBu_r"
+    plot_surf_roi(mesh, roi_map=roi_map,
+                  hemi='left',
+                  colorbar=False,
+                  engine="plotly",
```

Comment:
Sorry, is the issue specific to plotly ?

Reply:
let me check

Reply:
good catch: the issue also happens with matplotlib